fem/integ/bilininteg_diffusion_kernels.hpp

```diff
@@ -633,6 +634,8 @@ inline void SmemPADiffusionApply2D(const int NE,
                                    const bool symmetric,
                                    const Array<real_t> &b_,
                                    const Array<real_t> &g_,
+                                   const Array<real_t> &bt_,
+                                   const Array<real_t> &gt_,
                                    const Vector &d_,
                                    const Vector &x_,
                                    Vector &y_,
```

Comment:
These are necessary so that the "fallback" or non shared memory kernels match the signature of the shared memory implementations.  I will make these parameters anonymous because they are unused.

Reply:
I think it's better not to make the anonymous, and to keep the name. `MFEM_CONTRACT_VAR` and a comment in the body should be enough to explain this.

This is already done elsewhere in the code, e.g. `SmemPAMassApply2D` and `SmemPAMassApply3D`.

If the implementation of the shared memory kernel changes, they might use the transposed versions (it's just an implementation detail that they are not used currently).